### PR TITLE
Publish but do not install tests

### PR DESCRIPTION
**Pull request by polyzen**
Also remove unused import and touch up formatting.

### CODE REVIEW

**Comment by polyzen** [comment]
Doesn't do the job.

**Comment by polyzen** [comment]
Alright, looks like I bit off more than I can chew here.

File: setup.py [hunk 1]
```diff
@@ -21,38 +21,39 @@
...
+    author_email="jelmer@jelmer.uk",
+    license="GNU GPLv3 or later",
+    url="https://www.xandikos.org/",
+    install_requires=[
+        'icalendar',
+        'dulwich>=0.19.1',
+        'defusedxml',
+        'jinja2',
+    ],
+    packages=find_packages(exclude=['xandikos/tests']),
```
Comment (jelmer) [inline setup.py]: I think exclude takes a filename pattern rather than a path, so you'd want exclude=["tests"]

**Comment by polyzen** [comment]
On November 21, 2018 7:50:25 AM EST, "Jelmer Vernooĳ" <notifications@github.com> wrote:
>jelmer commented on this pull request.
>
>
>
>> -          'Operating System :: POSIX',
>-      ])
>+    description="Lightweight CalDAV/CardDAV server",
>+    long_description=long_description,
>+    version=version,
>+    author="Jelmer Vernooij",
>+    author_email="jelmer@jelmer.uk",
>+    license="GNU GPLv3 or later",
>+    url="https://www.xandikos.org/",
>+    install_requires=[
>+        'icalendar',
>+        'dulwich>=0.19.1',
>+        'defusedxml',
>+        'jinja2',
>+    ],
>+    packages=find_packages(exclude=['xandikos/tests']),
>
>I think exclude takes a filename pattern rather than a path, so you'd
>want exclude=["tests"]
>
>-- 
>You are receiving this because you modified the open/close state.
>Reply to this email directly or view it on GitHub:
>https://github.com/jelmer/xandikos/pull/84#pullrequestreview-177222302

Sorry, missed that.

-- 
Best,
polyzen

**Pull request closed without merging**